front/src/common/RollingStockSelector/RollingStockCard.jsx

```diff
@@ -55,7 +55,7 @@ function RollingStockCard(props) {
       onClick={displayCardDetail}
       tabIndex={0}
       ref={ref2scroll}
-      data-testid={`rollingstock${data.id}`}
```

Comment:
Why not keep a more meaningful data-testid with "rollingstock" before the data.name.
`data-testid={rollingstock-${data.name}}`